BasicSample/Assets/ARPlane/Materials/Plane_Ceiling.mat

```diff
@@ -11,7 +11,7 @@ Material:
   m_Shader: {fileID: 46, guid: 0000000000000000f000000000000000, type: 0}
   m_ShaderKeywords: _ALPHAPREMULTIPLY_ON
   m_LightmapFlags: 4
-  m_EnableInstancingVariants: 0
+  m_EnableInstancingVariants: 1
```

Comment:
curious on why this is not enabled before

Reply:
This is to enable instanced rendering for plane's prefab for plane finding scenario.